preprocessors/tif-decompress.preprocessor.js

```diff
@@ -0,0 +1,43 @@
+var gdal = require('gdal');
+
+module.exports = function(infile, outfile, callback) {
+  var ds;
+  var dscopy;
+  var driver = gdal.drivers.get('GTiff');
+  outfile = outfile + '.tif';
```

Comment:
Ah, thanks for catching @springmeyer . It doesn't actually need an extension. And it makes sense that outfile is different in the final output, since my changes within the decompress processor wouldn't carry through to the callback. I'm removing the added `.tif`